src/features/tasks/tasksSaga.ts

```diff
@@ -19,7 +19,7 @@ function* fetchExampleTasksHandler() {
   }
 }
 
-function* saveTasksInLocalStorageHandler() {
+function* saveTasksInLocalStorageHandler(): any {
```

Comment:
there is a type you can use for Generator functions - Generator<never, void, unknown> in which you can specify the generics which it returns